chat-sample/package.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "chat-agent-sample",
+    "name": "chat-sample",
```

Comment:
Looks like you need to run `npm install`. The package-lock.json still has the old values.